src/lm_buddy/jobs/configs/base.py

```diff
@@ -0,0 +1,40 @@
+import contextlib
+import tempfile
+from pathlib import Path
+
+from pydantic_yaml import parse_yaml_file_as, to_yaml_file
+
+from lm_buddy.types import BaseLMBuddyConfig
+
```

Comment:
define an `__all__ = [...]` for your new files

Reply:
Ill add the __all__ to the configs folder.